features/exploredash/src/main/java/org/dash/wallet/features/exploredash/ExploreSyncWorker.kt

```diff
@@ -54,41 +54,52 @@ class ExploreSyncWorker @AssistedInject constructor(
 
         var localDataTimestamp = 0L
         var remoteDataTimestamp = 0L
+        var preloadedDbTimestamp = 0L
+
         try {
             syncStatus.setSyncProgress(0.0)
 
             val timeInMillis = measureTimeMillis {
-                if (exploreRepository.localTimestamp <= 0L) {
+                val updateFile = exploreRepository.getUpdateFile()
+                exploreRepository.preloadFromAssetsInto(updateFile)
+                preloadedDbTimestamp = exploreRepository.getTimestamp(updateFile)
+
+                log.info("preloaded data timestamp: $preloadedDbTimestamp (${Date(preloadedDbTimestamp)})")
+
+                if (exploreRepository.localTimestamp == 0L ||
+                    exploreRepository.localTimestamp < preloadedDbTimestamp
+                ) {
                     // force data preloading for fresh installs
+                    // and a newer preloaded DB
                     AppExploreDatabase.getAppDatabase(
                         appContext,
                         config,
                         exploreRepository
                     )
-                }
-
-                localDataTimestamp = exploreRepository.localTimestamp
-                log.info("local data timestamp: $localDataTimestamp (${Date(localDataTimestamp)})")
+                } else {
```

Comment:
If the prepackaged database is newer than the local one, it will be unpacked and the GCloud database won't be checked until the next launch. 
This is fine - we cannot rely on the localDataTimestamp check here until the next launch anyway since it is updated asynchronously after DB is unpacked and opened.